autoprotocol/protocol.py

```diff
@@ -3767,6 +3767,154 @@ def uncover(self, ref, store_lid=None):
+            Front and rear window extension.
+        remove_coincident_events : bool, optional
+            Remove coincident events.
+
+        Returns
+        -------
+        FlowCytometry
+            Returns a :py:class:`autoprotocol.instruction.FlowCytometry`
+            instruction created from the specified parameters.
+
+        Raises
+        ------
+        TypeError
+            If `lasers` is not of type list.
+        TypeError
+            If `samples` is not of type Well, list of Well, or WellGroup.
+        TypeError
+            If `width_threshold` is not a number.
+        TypeError
+            If `window_extension` is not a number.
+        TypeError
+            If `remove_coincident_events` is not of type bool.
+
+        Examples
+        --------
+        Example flow cytometry protocol
+
+        .. code-block:: python
+
+            p = Protocol()
+            plate = p.ref("sample-plate", cont_type="384-flat", discard=True)
+
+            lasers = [FlowCytometry.builders.laser(
+                excitation="405:nanometers",
+                channels=[
+                    FlowCytometry.builders.channel(
+                        emission_filter=FlowCytometry.builders.emission_filter(
+                            channel_name="VL1",
+                            shortpass="415:nanometers",
+                            longpass="465:nanometers"
+                        ),
+                        detector_gain="10:millivolts"
+                    )
+                ]
+            )]
+
+            collection_conds = FlowCytometry.builders.collection_conditions(
+                acquisition_volume="5.0:ul",
+                flow_rate="12.5:ul/min",
+                wait_time="10:seconds",
+                mix_cycles=10,
+                mix_volume="10:ul",
+                rinse_cycles=10
+            )
+
+            p.flow_cytometry("flow-1234", plate.wells_from(0, 3), lasers,
+                             collection_conds)
+
+        Autoprotocol Output:
+
+        .. code-block:: json
+
+            {
+              "op": "flow_cytometry",
+              "dataref": "flow-1234",
+              "samples": [
+                "sample-plate/0",
+                "sample-plate/1",
+                "sample-plate/2"
+              ],
+              "lasers": [
+                {
+                  "excitation": "405:nanometer",
+                  "channels": [
+                    {
+                      "emission_filter": {
+                        "channel_name": "VL1",
+                        "shortpass": "415:nanometer",
+                        "longpass": "465:nanometer"
+                      },
+                      "detector_gain": "10:millivolt"
+                    }
+                  ]
+                }
+              ],
+              "collection_conditions": {
+                "acquisition_volume": "5:microliter",
+                "flow_rate": "12.5:microliter/minute",
```

Comment:
I think that for internal consistency we're going to make this `flowrate` even though the accepted spelling seems to be `flow rate`.